src/ListComponent.js

```diff
@@ -17,15 +17,15 @@ const ListComponent = (props) => {
     const [list, setList] = useState([]);
     const [showCal, setShowCal] = useState(false);
     const [calDate, setCalDate] = useState(new Date());
-    const [done, complete] = useState(true);
 
     const addTask = () => { 
         if (input !== "") { 
             const taskToAdd = { 
                 id: Math.random(),
                 title: input,
                 date: calDate,
-                details: ''
+                details: '',
+                completed: false,
```

Comment:
@chloedeveloper, we basically needed to add state management to see if an item is completed for each individual item.

Reply:
This was my comment/improvement on the previous PR, seems logical.